test/pagination/offset-paging.js

```diff
@@ -777,3 +777,54 @@ test('should handle an interface type', async t => {
   }
   t.deepEqual(expect, data.user.writtenMaterial)
 })
+
+test('should not allow pagination with greater than pageSizeLimit at the root', async t => {
+  const query = makeUsersQuery({ first: 1000 })
+  const { data, errors } = await run(query)
+
+  t.deepEqual(data.users, null)
+  t.deepEqual(errors[0].message, 'Maximum page size of User type, is 100')
```

Comment:
I think that comma shouldn't be there right? `'Maximum page size of User type, is 100' -> 'Maximum page size of User type is 100`

Reply:
Removed comma from both error thrown and the test.